docs/multicluster.md

```diff
@@ -205,18 +205,18 @@ With our clusters infrastructure templates created, we can now apply the templat
 
 ```bash
 $ cd east
-$ terraform init -var-file=spk.tfvars -backend-config=./backend.tfvars
-$ terraform plan -var-file=spk.tfvars
-$ terraform apply -var-file=spk.tfvars
+$ terraform init -var-file=bedrock.tfvars -backend-config=./backend.tfvars
+$ terraform plan -var-file=bedrock.tfvars
+$ terraform apply -var-file=bedrock.tfvars
 ```
 
 This deploys our `east` cluster.  We can naturally do the same thing for our `west` cluster with the same set of commands:
 
 ```bash
 $ cd west
-$ terraform init -var-file=spk.tfvars -backend-config=./backend.tfvars
-$ terraform plan -var-file=spk.tfvars
-$ terraform apply -var-file=spk.tfvars
+$ terraform init -var-file=bedrock.tfvars -backend-config=./backend.tfvars
```

Comment:
Has the bedrock cli been updated to create bedrock.tfvars instead of spk.tfvars?

Reply:
@jmspring  yes